src/Arcus.Templates.AzureFunctions.Databricks.JobMetrics/Startup.cs

```diff
@@ -20,39 +19,50 @@ public override void ConfigureAppConfiguration(IFunctionsConfigurationBuilder bu
         {
             builder.ConfigurationBuilder.AddEnvironmentVariables();
         }
-
+        
         // This method gets called by the runtime. Use this method to add services to the container.
         // For more information on how to configure your application, visit https://docs.microsoft.com/en-us/azure/azure-functions/functions-dotnet-dependency-injection
         public override void Configure(IFunctionsHostBuilder builder)
         {
             IConfiguration config = builder.GetContext().Configuration;
-
+            
             builder.ConfigureSecretStore(stores =>
             {
 //[#if DEBUG]
                 stores.AddConfiguration(config);
 //[#endif]
-
+                
                 stores.AddEnvironmentVariables();
-
+                
```

Comment:
Unnecessary whitespace :)

Reply:
Not exactly. The dotnet template sometimes removes the lines without characters, so the result is an all pushed together in a not clean code file. The whitespace makes sure that we don't have that.